.github/workflows/test.yaml

```diff
@@ -36,12 +36,15 @@ jobs:
         run: |
           python setup.py develop --install-dir ~/.cache/site-packages
           pip install pytest
-      - name: Run tests
+      - name: Run integration tests
         working-directory: ./tests
         run: pytest test*.py --durations=0 # Run actual tests
       - name: Run unit tests
         working-directory: ./tests/unittests
         run: pytest test*.py --durations=0 # Run actual tests
-      - name: Run other examples
-        working-directory: ./
-        run: pytest ./examples/*.py ./covasim/cruise_ship/test*.py
+      - name: Run cruise ship tests
+        working-directory: ./covasim/cruise_ship
+        run: pytest test*.py
+      - name: Run examples
+        working-directory: ./examples
+        run: pytest *.py
```

Comment:
I swear Google is gaslighting me here because I can't find any mention of this, but pytest used to raise an error if it didn't find any tests, which is the case for these folders. I swear.

Reply:
Okay, I'm not crazy. (Or perhaps I should say, my memory isn't that faulty). Looks like they recently changed this because nonzero exit code was definitely making CI tests fail a couple weeks ago. https://github.com/pytest-dev/pytest/issues/2393

Reply:
@lgeorge-idm Wait, no, I'm not crazy at all. They're still failing on this PR. 🤣 Well, this is why. It's stupid and I consider it to be a bug in pytest (`sh -c 'pytest testsuite1/ {posargs}; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'`!? no thanks), but I couldn't find any way around it.

Reply:
I forgot to set the flag on this correctly. Fixing now.

Reply:
There's a flag?! I looked for one...

Reply:
Fixed this. It was a combo if moving the run stuff into a function and specifiying the python_functions pattern matching the run function in each file and `capture=tee-sys`.